src/App.jsx

```diff
@@ -1,22 +1,43 @@
 import * as React from 'react'
-import Pagination from './Pagination'
-import Button from './Button'
 import Card from './Card'
+import Navigation from './Navigation'
+import cardData from './questen-data.json'
+import { v4 as uuidv4 } from 'uuid'
 
 export default () => {
+  const cardDataWithIDs = cardData.map(data => ({
+    ...data,
+    id: uuidv4(),
+  }))
+
+  const navItems = [
+    { title: 'Home', isActive: true },
+    { title: 'About' },
+    { title: 'Imprint' },
+  ]
   return (
     <div>
-      <Button isActive>Active</Button>
-      <Button>not Active</Button>
-      <Pagination totalPages={34} currentPage={1} />
-      <Pagination totalPages={34} currentPage={2} />
-      <Pagination totalPages={34} currentPage={34} />
-      <Card text="ich bin ein kleiner Text" title="Titel" />
-      <Card
-        isBookmarkActive
-        text="ich bin anderer ein kleiner Text"
-        title="Titel 2"
-      />
+      <Navigation navItems={navItems} />
+      {cardDataWithIDs.map(element => {
```

Comment:
you can destructure the properties here and also, it'd be better to write `correct-answer` in camelCase as we normally do in JavaScript. Just for readability.
```suggestion
      {cardDataWithIDs.map(({ id, question, correctAnswer, incorrectAnswer, isAnswerHidden, isBookmarkActive } => {
```